recipes.js

```diff
@@ -1,10 +1,24 @@
-const mongoose = require('mongoose');
-const Schema   = mongoose.Schema;
-const data = require('./data.js');
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
 
-mongoose.connect('mongodb://localhost/recipeApp')
-  .then(() => {
-    console.log('Connected to Mongo!');
-  }).catch(err => {
-    console.error('Error connecting to mongo', err);
-  });
+const recipeSchema = new Schema({
+  title: { type: String, required: true },
+  level: {
+    type: String,
+    enum: ["Easy Peasy", "Amateur Chef", "UltraPro Chef"]
+  },
+  ingredient: [],
+  cuisine: { type: String, required: true },
+  dishType: { type: String },
+  image: {
+    type: String,
+    match: /^https?:\/\//
+  },
+  duration: { type: Number, min: 0 },
+  creator: { type: String },
+  created: [Date]
```

Comment:
to set the default value of when it's created to the current time you can add `default : Date.now `